### PR TITLE
Show sender phone number instead of Unknown sender in Reports Content.

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/medic-webapp#4231

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Hi @garethbowen. Please review. Thanks!

File: translations/messages-en.properties [hunk 1]
```diff
@@ -979,5 +979,5 @@ instance.upgrade.confirm.warning.text = This action cannot be undone, make sure
 confirm.delete.progress = Deletion in progress
 confirm.delete.totals = {{totalDeleted}}/{{totalSelected}} deleted...
 messages.unknown.sender = Unknown sender
-messages.sent.by = Sent by {{senderName}}
+messages.sent.by = Submitted by {{senderName}}
```
Comment (garethbowen) [inline translations/messages-en.properties]: Unfortunately this won't work - the next time the messages are exported from our collaboration tool the default values will be overwritten. Follow [this documentation](https://github.com/medic/medic-docs/blob/master/development/translations.md#modifying-any-existing-translation-values) to update the messages directly in POE. If you don't have access to POE get a login from @abbyad .

**Comment by garethbowen** [comment]
Looks good apart from one little thing...

File: translations/messages-en.properties (hunk 1 above)
Comment (dianabarsan) [reply translations/messages-en.properties]: I've reverted the translation change and edited the tests so they are translation-independent.

Comment (garethbowen) [reply translations/messages-en.properties]: Would you like me to update POE and export so we can get this one merged?

**Pull request merged**